downstream/modules/platform/proc-renew-ssl-cert.adoc

```diff
@@ -0,0 +1,17 @@
+[id="renew-ssl-cert_{context}"]
+
+= Renewing the self-signed SSL certificate
+
+[role="_abstract"]
+The following steps regenerate a new SSL certificate for both {ControllerName} and {HubName}.
+
+.Procedure
+. Add `aap_service_regen_cert=true` to the inventory file in the `[all:vars]` section:
++
+----
+[all:vars]
+aap_service_regen_cert=true
+----
+. In the inventory file, comment out the the hosts services sections that do not require the ssl certificate renewal.
```

Comment:
Is it possible to leave this out? I understand the intent but if AAP is managing the self-signed certificate, then it really shouldn't matter if it renews the SSL certificate for the other components.

Reply:
More specifically, the linke about the hosts services; not the `[all:vars]`.

Reply:
@demonpig 
Do you mean that I should delete the "In the inventory file, comment out the the hosts services sections that do not require the ssl certificate renewal" line?

Reply:
Yes, delete the line entirely.